conference.js

```diff
@@ -1390,6 +1447,16 @@ export default {
             promise = this.useVideoStream(null);
         }
 
+        // mute the presenter track if it exists.
+        if (this.localPresenterVideo) {
+            APP.store.dispatch(
+                setVideoMuted(true, MEDIA_TYPE.PRESENTER));
```

Comment:
We might need to add a little something here. When muting the presenter, viewers will see the video flickering from screen + video to nothing back to screen. By looking at the thumbnail I noticed that the receivers *think* the video is completely muted (we send it over siognalling), but we shouldn't in this case.

I'm leaving this comment here because I think it's in context, not necessarily because the fix must happen here.

Reply:
This because of how setEffect is implemented. We remove the stream (w/ efffect) and then add the same stream (w/o effect) back to the conference. The flickering will go away when we switch to using RTCRtpSender.replaceTrack().

Reply:
Okidoki!